front/src/applications/operationalStudies/components/ManageTrainSchedule/Itinerary/DisplayVias.tsx

```diff
@@ -25,7 +25,7 @@ function InputStopTime({ index, setIndexSelected, dispatchAndRun }: InputStopTim
   const vias = useSelector(getVias);
   const [stopTime, setStopTime] = useState(vias[index].duration ? vias[index].duration : 0);
   const [firstStart, setFirstStart] = useState(true);
-  const debouncedStopTime = useDebounce(stopTime, 1000);
+  const debouncedStopTime = useDebounce(stopTime!, 1000) as number;
```

Comment:
You need to remove the !, and to modify stopTime like this:
```
    const [stopTime, setStopTime] = useState(vias[index].duration || 0);
```

then

```suggestion
  const debouncedStopTime = useDebounce(stopTime, 1000);
```

Reply:
done